x-pack/platform/plugins/shared/automatic_import/server/graphs/ecs/pipeline.ts

```diff
@@ -219,7 +225,10 @@ export function createPipeline(state: EcsMappingState): Pipeline {
   if (state.additionalProcessors.length > 0) {
     ingestPipeline = combineProcessors(ingestPipeline, state.additionalProcessors);
   }
-  return ingestPipeline;
+  return {
```

Comment:
After some discussion, let's also ensure we successfully process the samples (and fails otherwise)